app/models/pet.rb

```diff
@@ -132,7 +132,7 @@ def self.ransackable_scopes(auth_object = nil)
   end
 
   def self.ransack_adopted(boolean)
-    boolean ? adopted : unadopted
+    boolean ? adopted : Pet.where.not(id: Pet.adopted)
```

Comment:
Sorry, I see you have already done this! Can we make this a scope called `unadopted` instead?

Reply:
I can do that once I get the tests passing! Still having problems with this. 😅

Reply:
Done :)